frontend/src/scenes/events/EventsTable.js

```diff
@@ -190,10 +160,110 @@ function _EventsTable({ fixedFilters, filtersEnabled = true, pageKey }) {
             },
         },
     ]
+    const otherEventProperties = eventProperties.filter(
+        (e) => defaultColumns.find((d) => d.eventProperties && d.eventProperties.includes(e.value)) === undefined
+    )
+    const availableConfigsOptions = [
+        ...defaultColumns.map(tableColumnToCheckboxOption),
+        ...otherEventProperties.map(eventPropertyToCheckboxOption),
+    ]
+    let selectedConfigOptions = columnConfig === 'DEFAULT' ? defaultColumns.map((e) => e.key) : columnConfig
+
+    const columnsToRenderFromConfig =
+        columnConfig === 'DEFAULT'
+            ? defaultColumns
+            : columnConfig.map(
+                  (e) =>
+                      defaultColumns.find((d) => d.key === e) || {
+                          title: keyMapping['event'][e] ? keyMapping['event'][e].label : e,
+                          key: e,
+                          render: function renderURL({ event }) {
+                              if (!event) {
+                                  return { props: { colSpan: 0 } }
+                              }
+                              if (filtersEnabled) {
+                                  return (
+                                      <FilterPropertyLink
+                                          className={'ph-no-capture ' + rrwebBlockClass}
+                                          property={e}
+                                          value={event.properties[e]}
+                                          filters={{ properties }}
+                                      />
+                                  )
+                              }
+                              return <Property value={event.properties[e]} />
+                          },
+                          ellipsis: true,
+                      }
+              )
+    let columns = [
+        {
+            title: `Event${eventFilter ? ` (${eventFilter})` : ''}`,
+            key: 'event',
+            fixed: 'left',
+            width: 100,
+            render: function renderEvent(item) {
+                if (!item.event) {
```

Comment:
This row now doesn't have any styling anymore. Also, when you add more columns it doesn't go full width:
![image](https://user-images.githubusercontent.com/1727427/104178556-d67f3880-540a-11eb-8a00-4e9458f6f8dd.png)

Reply:
which row ? the row between heading and data ? or the heading @timgl  ?

Reply:
Fixed the row between heading and event data in 47045c721360b5089272da22e6ccb84ba25ff33c .